althea_kernel_interface/src/create_wg_key.rs

```diff
@@ -21,21 +21,21 @@ impl KernelInterface {
     }
 
     pub fn create_wg_keypair(&self) -> Result<[String; 2], Error> {
-        let mut genkey = Command::new("wg")
+        let genkey = Command::new("wg")
             .args(&["genkey"])
             .stdout(Stdio::piped())
             .output()
             .unwrap();
 
-        let mut genstdout = genkey.stdout;
+        let genstdout = genkey.stdout;
```

Comment:
Is a 4-character difference in 2 uses worth a separate variable for this?

Reply:
not really, fixed